modules/web/src/main/scala/Application.scala

```diff
@@ -38,6 +42,21 @@ object Application {
         val limit   = n.getOrElse(100)
         gs.queryProgramsByName(pattern, limit).flatMap(ps => Ok(ps.map(p => (p._1, p._2)).asJson ))
 
+      // Import an observation from the legacy ODB.
+      case GET -> Root / "api" / "import" / "obs" / o :? Host(h) as gs =>
```

Comment:
i wonder if these endpoints should be tested

Reply:
Yeah it would be good to add tests … maybe in another pr?

Reply:
I also think we need to look into the Rho DSL for defining endpoints because it generates swagger doc. I'm concerned that we don't have a good way to document the API.

Reply:
GraphQL maybe a better approach and it support generated documentation by default